docs/user/pixel_observations.md

```diff
@@ -0,0 +1,122 @@
+- Unflattens observations when necessary to preserve relationships between spatial data.
+
+
+### Pixel-specific Wrappers
+These are used to modify the environment observations, usually in ways that would reduce the input size and improve training speed. They can be found in the `garage/envs/wrappers` directory. The ones to note here are:
+
+#### PixelObservationWrapper
+This wraps around environments that typically produce observations that are not images, and instead generates pixel observations from the wrapped environment. The behavior of this wrapper is similar to that of `gym.wrappers.pixel_observation.PixelObservationWrapper`, except that the gym wrapper returns a `gym.spaces.Dict` observation space:
+
+```python
+from gym.wrappers.pixel_observation import PixelObservationWrapper as gymWrapper
+from garage.envs.wrappers import PixelObservationWrapper as garageWrapper
+
+env = gym.make("Pendulum_v0")
+# env.observation_space = Box(3,)
+env.reset() # needed when using gym's wrapper
+gym_wrapped = gymWrapper(env)
+# gym_wrapped.observation_space = Dict(pixels:Box(1000, 1000, 3))
+garage_wrapped = garageWrapper(env)
+# garage_wrapped.observation_space = Box(1000, 1000, 3)
+```
+
+Notice that the resulting observation space is 3-dimensional, and now represents the RGB pixel values.
+
+ #### Resize
+ This wrapper is typically used to scale down an image. Note that *size* here refers to its resolution, so images are *not* cropped. Only 2D single-channel images can be resized, so this wrapper is usually applied after the `Grayscale` wrapper.
+
+```python
+env = gym.make('Env')
+# env.observation_space.shape = (100, 100)
+env_wrapped = Resize(env, 64, 64) # env, width, height
+# env_wrapped.observation_space.shape = (64, 64)
+```
+#### Grayscale
+
+As the name implies, this wrapper converts a 2D RGB image into a single-channel, grayscale image. This can only be applied to 3-channel images.
+
+```python
+env = gym.make('Env')
+# env.observation_space.shape = (100, 100, 3)
+env_wrapped = Grayscale(env)
+# env_wrapped.observation_space.shape = (100, 100)
+```
+#### StackFrames
+
+Frame stacking is a technique used in RL to create observations from which temporal data can be inferred. As a concrete example, consider the `Pendulum-v0` environment where the agent must balance the pendulum vertically by applying some amount of torque to the shaft. To do this reliably, the agent needs to know both the current position of the pendulum and the direction in which it is moving. The former is given by the current image observation, but the latter cannot be inferred from one image alone. To combat this, you can stack N (say, 3) images into one , which would embed this information into one observation.
+
+Frames are stacked on a rolling basis. When stacking N frames, the N most recent frames are returned as the observation.
```

Comment:
I think it would be good to mention that stacking increases the number of *channels* of the observation (it doesn't make the image taller or wider).

Reply:
Good call, done.